.github/workflows/merge-to-stage.js

```diff
@@ -224,7 +224,10 @@ const main = async (params) => {
     throw new Error('ADD NEW RCPs');
   }
   for (const { start, end } of RCPDates) {
-    if (start <= now && now <= end) {
+    const adjustedStart = new Date(start);
+    adjustedStart.setDate(adjustedStart.getDate() - 1);
```

Comment:
As discussed, let's do 2, one to get SOT approvals, one to actually merge.

Also, I think most RCPs start on Sundays, even this approach might not be enough and we should instead look for the Wednesday before an RCP, so that we can eventually merge it on Thursday. But that may complicate things even further 🙂

Reply:
The alternative is looking into the RCP dates and keeping a seperate list for stage and manually going through all the dates and editing them.

Contributions welcome 😁